zio-http/src/main/scala/zhttp/html/Elements.scala

```diff
@@ -250,6 +250,9 @@ trait Elements {
 }
 
 object Element {
+  val selfClosingTags =
+    List(area, base, br, col, embed, hr, img, input, link, meta, param, source, track, wbr).map(_.name)
```

Comment:
Taken from: http://xahlee.info/js/html5_non-closing_tag.html

Reply:
- Use a `Set` for better perf. 
- Use `String` and do a string comp